packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -250,6 +251,24 @@ export const SlackSingleChannelNotifier: NotificationIntegrationHelper<SlackNoti
   async integrationUpdated() {
     // Slack sends a system message on its own
     return 'success'
+  },
+
+  async standupResponseSubmitted(meeting, team, user, response) {
+    const options = {
+      searchParams: {
+        utm_source: 'slack standup submission',
+        utm_medium: 'product',
+        utm_campaign: 'standups',
```

Comment:
@tianrunhe mind giving a ✅ on the updated UTM params here?

Reply:
Tagging @avivapinchas for a review here

Reply:
Aviva [suggested](https://parabol.slack.com/archives/C03C3J42WDU/p1691109288833119?thread_ts=1691095523.822949&cid=C03C3J42WDU) to change the campaign to `notifications`

Reply:
@tianrunhe updated!